### ISSUE TITLE
Error on OAuth consent tab close

### ISSUE THREAD

**Issue by liyasthomas** [FabianLars/tauri-plugin-oauth#12]
It would be great if this plugin could throw an error when the OAuth consent screen tab is closed from the browser so that the listening server can prompt users the tab closed hence sign in again.

**Comment by FabianLars** [comment]
Good idea for sure! I have a lot on my plate right now though (ignoring that I'm not even at home rn) so I can't give you any ETA :)

Also, looking at how oauth providers love to _not_ follow the spec, this will probably be fun to implement 🙃
